hbase-client/src/main/java/org/apache/hadoop/hbase/client/RegionStatesCount.java

```diff
@@ -0,0 +1,167 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hbase.client;
+
+import org.apache.yetus.audience.InterfaceAudience;
+
+@InterfaceAudience.Private
+public final class RegionStatesCount {
+
+  private int openRegions;
+  private int splitRegions;
+  private int closedRegions;
+  private int regionsInTransition;
+  private int totalRegions;
+
+  private RegionStatesCount() {
+  }
+
+  public int getClosedRegions() {
+    return closedRegions;
+  }
+
+  private void setClosedRegions(int closedRegions) {
```

Comment:
nit:  let's try to align the get & set placement ?   all getXX first, then setXX ? or  getXX & setXX one by one ?

Reply:
Done @openinx 
Thanks